src/core/pile.sol

```diff
@@ -17,150 +17,213 @@ pragma solidity >=0.4.24;
 
 import "ds-note/note.sol";
 
-import { TitleOwned } from "tinlake-title/title.sol";
 
-contract TokenLike{
-    uint public totalSupply;
-    function balanceOf(address) public view returns (uint);
-    function transferFrom(address,address,uint) public;
-    function approve(address, uint) public;
+// ## PileLike the interface for any debt module.
+contract PileLike {
```

Comment:
Moved it out of pile for consistency. But it would make sense to move the interfaces into sep files. They should be independent of the contracts implementing, but always adding them to contracts using them generates a lot of duplicated code 🙈